nix/sources.json

```diff
@@ -0,0 +1,62 @@
+{
```

Comment:
I assume this will occasionally need to be updated. Can you document how to do so?